index.js

```diff
@@ -9,15 +9,25 @@ export default class Flatbush {
 
     /**
      * Recreate a Flatbush index from raw `ArrayBuffer` or `SharedArrayBuffer` data.
-     * @param {ArrayBuffer | SharedArrayBuffer} data
+     * @param {Uint8Array | ArrayBuffer | SharedArrayBuffer} data
      * @returns {Flatbush} index
      */
     static from(data) {
-        // @ts-expect-error duck typing array buffers
-        if (!data || data.byteLength === undefined || data.buffer) {
-            throw new Error('Data must be an instance of ArrayBuffer or SharedArrayBuffer.');
+        let byteOffset = 0;
+        if (data instanceof Uint8Array) {
```

Comment:
I'm wondering — would it be conceptually simpler and more compact to just do `from(data, byteOffset = 0)`, and leave view handling for the user?